### DISCUSSION TITLE
Panic Button

### DISCUSSION THREAD

**Discussion by TomekPL-1** [secretcowlevel/DoomtrooperPublic#65] [Card Balance]
This card is riddiculous.

I suggest to change it to something like:

Panic Button (GENERAL) [R]
EQUIPMENT
If this Warrior is a defending combatant, Annihilate the Panic Button to deploy a Warrior with lower base V from your hand with no Action, you still need to pay his [DP] cost. It becomes the defender.

**Comment by Sol-au-fein** [comment]
`deploy a Warrior with lower base V from your hand with no Action, `
How many actions you exactly have during opponents turn?

**Reply by TomekPL-1** [reply]
This is same like in current version of this card.
